cdap-ui/app/features/operations/ops-ctrl.js

```diff
@@ -178,8 +190,46 @@ angular.module(PKG.name+'.feature.dashboard')
       return {title : 'System metrics', columns : widgets};
     }
 
+    function startPolling (widget) {
+      dataSrc.poll({
```

Comment:
We have ```startPolling``` in ops-helper and in dashboardhelper .. in the next refactor will we be combining them into one?